data/.gitignore

```diff
@@ -1,7 +1,9 @@
 .env
 raw/acs/*.csv
-raw/shapefiles/*.zip
 intermediate/acs/*.csv
 
-# Extra shapefile data
+# Shapefiles
+raw/shapefiles/*.zip
 intermediate/shapefiles/*
+final/shapefiles/*
+final/acs/*
```

Comment:
The final ACS data and shapefiles are now so big that it's not feasible to store them under source control anymore. Instead, we'll have to require that new users run the Makefile to set up a dev environment.